tools/commissioner_thci/commissioner_impl.py

```diff
@@ -491,6 +494,8 @@ def _command(self, cmd, timeout=10):
                 response.append(line)
 
             self._sleep(0.1)
+            timeout -= 0.1
+
```

Comment:
The code look a lot like `_expect`. Why not extend `_expect` to support regression to reduce code duplication.

Reply:
Maybe leave this to future enhancement? since this is not mandatory/related to this bugfix PR.